### ISSUE TITLE
you cant deactivate it

### ISSUE THREAD

**Issue by Eli53223456776543234567876543234567** [uniphil/dom-destroyer#5]
its frustrating because somtimes i justwant to get rid of adds but when im donei cant deactivate it

**Comment by uniphil** [comment]
Hello @Eli53223456776543234567876543234567 !

It's a bug if you can't deactivate it. There are three things that should toggle it off:

- hitting `ESC` key
- clicking the icon from the toolbar again
- pressing the hotkey, CTRL(/cmd on mac) + SHIFT + L

Any of these should turn it off and put your page back to normal. Can you try and see if any of them work?

If it's still not turning off for you, please let me know what browser you're using and maybe we can figure out why :)

**Comment by Eli53223456776543234567876543234567** [comment]
No luck.
Eli

**Comment by Eli53223456776543234567876543234567** [comment]
i am using a chromebook

Eli
